Code/Engine/RendererFoundation/Device/Implementation/Device.cpp

```diff
@@ -642,6 +642,46 @@ ezGALTextureHandle ezGALDevice::CreateTexture(
   return ezGALTextureHandle();
 }
 
+ezResult ezGALDevice::ReplaceExisitingNativeObject(ezGALTextureHandle hTexture, void* pExisitingNativeObject)
+{
+  ezGALTexture* pTexture = nullptr;
+  if (m_Textures.TryGetValue(hTexture, pTexture))
+  {
+    for (auto it = pTexture->m_RenderTargetViews.GetIterator(); it.IsValid(); ++it)
```

Comment:
You need to do the same for m_ResourceViews and m_UnorderedAccessViews

Reply:
Will do!